src/tooltip/tooltip.js

```diff
@@ -76,6 +78,8 @@ MaterialTooltip.prototype.handleMouseLeave_ = function(event) {
 
   event.stopPropagation();
   this.element_.classList.remove(this.CssClasses_.IS_ACTIVE);
+  window.removeEventListener('scroll', this.boundMouseLeaveHandler);
+  window.removeEventListener('touchmove', this.boundMouseLeaveHandler, false);
```

Comment:
Move > Copy.

Moving would mean breaking the fix itself. Copy will simply ensure that they are removed when downgraded in-case missed at some other point.